pallets/author-noting/Cargo.toml

```diff
@@ -9,8 +9,10 @@ version = "0.1.0"
 [package.metadata.docs.rs]
 targets = [ "x86_64-unknown-linux-gnu" ]
 [dependencies]
+frame-benchmarking = { workspace = true, optional = true }
 frame-support = { workspace = true }
 frame-system = { workspace = true }
+hex = { workspace = true, features = [ "alloc" ] }
```

Comment:
this probably can be optional, we only use it in runtime-benchmarks right?¿

Reply:
Yes, I only use it for creating and reading the mock_proof.rs. So it is needed for `any(runtime-benchmarks, test)`.

Reply:
ok make it optional then and we import it with std and runtime-benchmarks